### PR TITLE
create input

### PR DESCRIPTION
**Pull request by gulli301185**

### CODE REVIEW

File: src/components/UI/input/InputPage.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+   borderRadius: 8,
+   '&:hover': {
+      outline: '1px solid #959595',
+   },
+
+   '&:Active': {
+      outline: '1px solid #048741',
+   },
+   '&:error': { outline: '1px solid #F91515' },
+})
```
Comment (erkinovakurmanzhan1) [inline src/components/UI/input/InputPage.jsx:29]: checked

**Comment by RussoOrmon** [comment]
I don't have any error

**Review by RussoOrmon — COMMENTED** [review]
I don't have any error

File: src/components/UI/input/InputPage.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { TextField } from '@mui/material'
+import styled from '@emotion/styled'
+
+const InputPage = ({ color, size, variant, placeholder, ...rest }) => {
```
Comment (songhee24) [inline src/components/UI/input/InputPage.jsx:5]: просто Input может ?

Comment (songhee24) [reply src/components/UI/input/InputPage.jsx:5]: и да нет ref

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**